gateway/src/api/latest.rs

```diff
@@ -103,161 +93,6 @@ impl StatusResponse {
     }
 }
 
-#[instrument(skip(service))]
-#[utoipa::path(
-    get,
-    path = "/projects/{project_name}",
-    responses(
-        (status = 200, description = "Successfully got a specific project information.", body = shuttle_common::models::project::Response),
-        (status = 500, description = "Server internal error.")
-    ),
-    params(
-        ("project_name" = String, Path, description = "The name of the project."),
-    )
-)]
-async fn get_project(
-    State(RouterState { service, .. }): State<RouterState>,
-    ScopedUser { scope, .. }: ScopedUser,
-) -> Result<AxumJson<project::Response>, Error> {
-    let state = service.find_project(&scope).await?.into();
-    let response = project::Response {
-        name: scope.to_string(),
-        state,
-    };
-
-    Ok(AxumJson(response))
-}
-
-#[utoipa::path(
-    get,
-    path = "/projects",
-    responses(
-        (status = 200, description = "Successfully got the projects list.", body = [shuttle_common::models::project::Response]),
-        (status = 500, description = "Server internal error.")
-    ),
-    params(
-        PaginationDetails
-    )
-)]
-async fn get_projects_list(
-    State(RouterState { service, .. }): State<RouterState>,
-    User { name, .. }: User,
-    Query(PaginationDetails { page, limit }): Query<PaginationDetails>,
-) -> Result<AxumJson<Vec<project::Response>>, Error> {
-    let limit = limit.unwrap_or(u32::MAX);
-    let page = page.unwrap_or(0);
-    let projects = service
-        // The `offset` is page size * amount of pages
-        .iter_user_projects_detailed(&name, limit * page, limit)
-        .await?
-        .map(|project| project::Response {
-            name: project.0.to_string(),
-            state: project.1.into(),
-        })
-        .collect();
-
-    Ok(AxumJson(projects))
-}
-
-#[instrument(skip_all, fields(%project))]
-#[utoipa::path(
-    post,
-    path = "/projects/{project_name}",
-    request_body = shuttle_common::models::project::Config,
-    responses(
-        (status = 200, description = "Successfully created a specific project.", body = shuttle_common::models::project::Response),
-        (status = 500, description = "Server internal error.")
-    ),
-    params(
-        ("project_name" = String, Path, description = "The name of the project."),
-    )
-)]
-async fn create_project(
```

Comment:
Maybe this will change into `deploy_project`? As I view it, it should coordinate the building & deploying process.

Reply:
Won't a deploy be redirected to deployer and builder, and then the gateway will know it's ready when it gets an address from deployer?